arch/arm/core/aarch32/cortex_r/irq_init.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2020 Stephanos Ioannidis <root@stephanos.io>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief ARM Cortex-R interrupt initialization
+ */
+
+#include <arch/cpu.h>
+#include <drivers/interrupt_controller/gic.h>
+
+/**
+ *
+ * @brief Initialize interrupts
+ *
+ * @return N/A
+ */
+void z_arm_int_lib_init(void)
```

Comment:
@ioannisg I will update this after #23421 is finalised

Reply:
ok